test/unit/au/org/emii/portal/AsyncDownloadControllerTests.groovy

```diff
@@ -19,7 +19,7 @@ class AsyncDownloadControllerTests extends ControllerUnitTestCase {
         controller.downloadAuthService = downloadAuthService.createMock()
 
         gogoduckService = mockFor(GogoduckService)
-        gogoduckService.demand.static.registerJob { params -> return "gogoduck_rendered_text" }
```

Comment:
Yeah, did work, but I think it messed up the other tests. It's being called once, so I think enough to mock it once...